elasticsearch/src/main/java/com/yahoo/ycsb/db/ElasticSearchClient.java

```diff
@@ -318,10 +333,13 @@ public Status update(String table, String key,
   public Status scan(String table, String startkey, int recordcount,
       Set<String> fields, Vector<HashMap<String, ByteIterator>> result) {
     try {
-      final RangeFilterBuilder filter = rangeFilter("_id").gte(startkey);
+      final RangeQueryBuilder filter = rangeQuery("_id").gte(startkey);
       final SearchResponse response = client.prepareSearch(indexKey)
-          .setTypes(table).setQuery(matchAllQuery()).setFilter(filter)
-          .setSize(recordcount).execute().actionGet();
+          .setTypes(table)
+          .setQuery(matchAllQuery())
+          .setSize(recordcount)
+          .execute()
+          .actionGet();
 
```

Comment:
I goofed here. ES no longer has filter, so .setFilter() is deprecated. 
with the new API, setQuery(matchAllQuery()).setFilter(filter) should be simplified to .setQuery(rangeQuery).
Good catch.